docs/code_ref/coordinates/index.rst

```diff
@@ -206,8 +206,8 @@ coordinates is::
    <SkyCoord (Helioprojective: obstime=2017-07-26T00:00:00.000, rsun=695700.0 km, observer=<HeliographicStonyhurst Coordinate for 'earth'>): (Tx, Ty) in arcsec
```

Comment:
> Shouldn't [line 54](https://github.com/sunpy/sunpy/pull/3782/files#diff-0ed4b40bbcab180c906609a092ecf502R54) and 58 say heliographic instead of heliocentric?

Ha!  Good catch.

> Should we also mention on this page (under section of [line 166](https://github.com/sunpy/sunpy/pull/3782/files#diff-0ed4b40bbcab180c906609a092ecf502R54)) that the observer is needed on Carrington's or it will be set as `None`?

I'd argue that there's no reason to over-stuff this section (which is just about accessing coordinate components).  I'll add a sentence in the observer section.